language-testutils/src/test/java/de/jplag/testutils/LanguageModuleTest.java

```diff
@@ -216,17 +217,17 @@ final void testTokenPositions(TokenPositionTestData testData) throws ParsingExce
             TokenType expectedType = this.languageTokens.stream().filter(type -> type.toString().equals(expectedToken.typeName())).findFirst()
                     .orElseThrow(() -> new IOException(String.format("The token type %s does not exist.", expectedToken.typeName())));
 
-            if (extractedTokens.stream().noneMatch(token -> token.getType() == expectedType && token.getLine() == expectedToken.lineNumber()
-                    && token.getColumn() == expectedToken.columnNumber() && token.getLength() == expectedToken.length())) {
+            if (extractedTokens.stream()
+                    .noneMatch(token -> token.getType() == expectedType && token.getStartLine() == expectedToken.startLine()
+                            && token.getStartColumn() == expectedToken.startColumn() && token.getEndLine() == expectedToken.endLine()
+                            && token.getEndColumn() == expectedToken.endColumn())) {
```

Comment:
Before this condition gets out of hand, we may just extract a variable ;)

Reply:
A function would surely be even better! There doesn't seem to be a `Token::equals` method as of now, but this would be a real use case then.